resources/views/components/reference.blade.php

```diff
@@ -1,16 +1,22 @@
-<figure class="{{$attributes->get('class')}} rounded-2xl bg-white p-6 shadow-lg ring-1 ring-gray-900/5 size-fit max-w-96 place-self-center">
+<figure class="{{ $attributes->get('class') }} rounded-2xl bg-white p-6 shadow-lg ring-1 ring-gray-900/5 size-fit max-w-96 place-self-center">
     <blockquote class="text-gray-900">
-        <p>{!! "“".$reference->description."”" !!}</p>
+        <p>{!! "“" . $reference->description . "”" !!}</p>
     </blockquote>
     <figcaption class="mt-6 flex items-center gap-x-4">
+        @php
+            $photoPath = (file_exists(storage_path('app/public/' . $reference->photo)) && !!$reference->photo)
+                         ? asset('storage/' . $reference->photo)
+                         : null;
+            $placeholder = $reference->sex === 'female'
+                           ? asset('graphics/placeholders/female.svg')
+                           : asset('graphics/placeholders/male.svg');
+        @endphp
```

Comment:
```suggestion
        @php
            $photoPath = (file_exists(storage_path('app/public/' . $reference->photo)) && !!$reference->photo)
                         ? asset('storage/' . $reference->photo)
                         : null;
            $placeholder = $reference->sex === 'female'
                           ? asset('graphics/placeholders/female.svg')
                           : asset('graphics/placeholders/male.svg');
        @endphp
```
Let's do that on backend